### DISCUSSION TITLE
Error occurred while executing Actions, unable to perform build operation

### DISCUSSION THREAD

**Discussion by networkabc** [rustdesk/rustdesk#10107] [Q&A]
As shown in the image, please help me identify what the issue is and how to resolve it. Thank you.

![2024-11-30 at 00 02 12](https://github.com/user-attachments/assets/3650fcec-a9ca-4d6f-8b64-c82fa9b8ac07)

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.
